File: NSPersistentContainer/NSPersistentContainerObjCSample/NSPersistentContainerObjCSample/SampleEntity+CoreDataProperties.m
//
//  SampleEntity+CoreDataProperties.m
//  NSPersistentContainerObjCSample
//
//  Created by nikolay.andonov on 10/10/16.
//  Copyright © 2016 nikolay.andonov. All rights reserved.
//

#import "SampleEntity+CoreDataProperties.h"

@implementation SampleEntity (CoreDataProperties)

+ (NSFetchRequest<SampleEntity *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"SampleEntity"];
}

@dynamic name;

@end

File: NSPersistentContainer/NSPersistentContainerSwiftSample/NSPersistentContainerSwiftSample/SampleEntity+CoreDataProperties.swift
//
//  SampleEntity+CoreDataProperties.swift
//  NSPersistentContainerSwiftSample
//
//  Created by nikolay.andonov on 10/10/16.
//  Copyright © 2016 nikolay.andonov. All rights reserved.
//

import Foundation
import CoreData


extension SampleEntity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<SampleEntity> {
        return NSFetchRequest<SampleEntity>(entityName: "SampleEntity");
    }

    @NSManaged public var name: String?

}
